### DISCUSSION TITLE
Request: A command to open the directory under the cursor to the opposite panel.

### DISCUSSION THREAD

**Discussion by chronakis** [doublecmd/doublecmd#1713] [Suggestions]
This is the feature I miss most and I haven't found it. A command (and a shortcut) that opens the directory under the cursor to the opposite pane. 

In totalcmd this was `Ctrl+right` when on the left pane and `Ctrl+left` when on the right pane.

The closest I found is the `cm_OpenDirInNewTab` but this is not what I want as it opens the tab in the same pane.

Thank you.

**Comment by JonnyTech** [comment]
> In totalcmd this was Ctrl+right when on the left pane and Ctrl+left when on the right pane.

These keyboard shortcuts work in DC too - as well as the internal commands `cm_transferleft` and `cm_transferright`.

Please post your version and system details (DC menu, help, about, copy to clipboard, paste here).

**Reply by chronakis** [reply]
Thank you so much! I searched everything that contains 'open', 'directory' or 'target' in the hotkeys and in the issues, feature requests, questions and documentation. You are absolutely right. The commands indeed exist but as I am on a mac, Ctrl+left/right arrow are navigating to left and right desktop. Command assigned to win (Cmd on mac) +left/right and works perfectly.

```
Double Commander
Version: 1.1.15 gamma
Revision: 505
Commit: 28125592a
Build date: 2024/06/12
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: x86_64-Darwin-cocoa
OS version: Mac OS X 14.5.0
```
